daemon/daemon.go

```diff
@@ -252,6 +253,8 @@ func (d *Daemon) updatePolicy(spec update.Spec, updates policy.Updates) DaemonJo
 			return metadata, nil
 		}
 
+		ctx = context.WithValue(ctx, "user", spec.Cause.User)
```

Comment:
It's perhaps arguable, but I think this falls foul of the advice "Use context Values only for request-scoped data that transits processes and APIs, not for passing optional parameters to functions." In other words, the author would be better as an argument to the specific git operations, rather than data smuggled through the context.

Reply:
In my mind this was request specific, which is why I used the context. I shall refactor.